MekHQ/src/mekhq/gui/model/UnitTableModel.java

```diff
@@ -125,16 +126,80 @@ public int getAlignment(int col) {
         };
     }
 
-    public @Nullable String getTooltip(int row, int col) {
-        Unit u = getUnit(row);
-        return switch (col) {
-            case COL_STATUS -> u.isRefitting() ? u.getRefit().getDesc() : null;
-            case COL_CREW_STATE -> u.getCrewState().getToolTipText();
-            case COL_QUIRKS -> u.getQuirksList();
+    /**
+     * Returns the tooltip for the specified row and column.
+     *
+     * @param rowIndex    the index of the row
+     * @param columnIndex the index of the column
+     * @return the tooltip for the specified row and column, or {@code null} if no tooltip is available
+     */
+    public @Nullable String getTooltip(int rowIndex, int columnIndex) {
+        Unit unit = getUnit(rowIndex);
```

Comment:
With tables... might want to give this a null guard :) (meaning, return null if unit is null)